qaframework-bdd-tests/src/test/resources/features/refapp-2.x/15-RolesAndPrivileges/RolesAndPrivileges.feature

```diff
@@ -0,0 +1,71 @@
+Feature: Roles & Privileges Management
+  
+  i want the user to Log in as an admin.Go to configure meta data.
+  Go to privileges, add a new privilege, edit it and delete the privilege
+  Go to roles,add a new role, edit it and delete a role.
+  lastly,Return to home page
```

Comment:
is this required ?

``js
i want the user to Log in as an admin.Go to configure meta data.
Go to privileges, add a new privilege, edit it and delete the privilege
Go to roles,add a new role, edit it and delete a role.
lastly,Return to home page
```